src/pybind/mgr/dashboard/frontend/src/app/shared/models/crud-table-metadata.ts

```diff
@@ -0,0 +1,17 @@
+import { TemplateRef } from '@angular/core';
+class TableColumn {
```

Comment:
we have some models there already like https://github.com/ceph/ceph/blob/main/src/pybind/mgr/dashboard/frontend/src/app/shared/models/cd-table-column.ts. can we reuse them or extend them here? @pereman2